### PR TITLE
add react_sum solution

### PR DESCRIPTION
**Pull request by DidenkoDenis306**
[DEMO LINK](https://DidenkoDenis306.github.io/react_sum/)

### CODE REVIEW

File: src/components/Sum.jsx [hunk 1]
```diff
@@ -1 +1,14 @@
-// export function Sum() {}
+import React from 'react';
+
+export const Sum = ({ a, b }) => (
+  <>
```
Comment (33Orange) [inline src/components/Sum.jsx:4]: The fragment '<>' can be removed, as it's not necessary when there is only one child element.

File: src/components/Sum.jsx [hunk 2]
```diff
@@ -1 +1,14 @@
...
+    <p>
+      {`Sum of ${a} and ${b} is ${a + b}`}
+    </p>
+  </>
```
Comment (33Orange) [inline src/components/Sum.jsx:8]: The closing fragment '</>' can be removed, as it's not necessary when there is only one child element.

File: src/App.jsx [hunk 3]
```diff
@@ -1,14 +1,23 @@
...
+      a={-5}
+      b={5}
+    />
+    <Sum
+      a={10}
+    />
+    <Sum
+      b={5}
+    />
+    <Sum />
```
Comment (33Orange) [inline src/App.jsx:21]: `[CODE STYLE] - If you have < 3 attribues on a tag and the values are short write the tag in one line (to make it easier to write and read).`
Checklist p1.
```suggestion
    <Sum a={2} b={3} />
    <Sum a={-5} b={5} />
    <Sum a={10} />
    <Sum b={5} />
    <Sum />
```

**Review by 33Orange — CHANGES_REQUESTED** [review]
Well done! 

A few moments with readability.

**Review by dinaminahi — APPROVED** [review]
Well done! 🥇

**Pull request closed without merging**